retroshare-gui/src/qss/retroclassic.qss

```diff
@@ -0,0 +1,454 @@
+	qproperty-backgroundColorFiltered: rgb(255, 240, 210);
+}
+
+FriendSelectionWidget
+{
+	qproperty-textColorOnline: blue;
+}
+
+GroupTreeWidget
+{
+	qproperty-textColorCategory: rgb(79, 79, 79);
+	qproperty-textColorPrivateKey: rgb(35,91,159);
+}
+
+MessagesDialog
+{
+	qproperty-textColorInbox: rgb(49, 106, 197);
+}
+
+NetworkDialog
+{
+	qproperty-backgroundColorSelf: yellow;
+	qproperty-backgroundColorOwnSign: rgb(69, 255, 69);
+	qproperty-backgroundColorAcceptConnection: rgb(67, 192, 67);
+	qproperty-backgroundColorHasSignedMe: rgb(178, 66, 178);
+	qproperty-backgroundColorDenied: lightGray;
+	qproperty-textColor: black;
+}
+
+NewFriendList
+{
+	qproperty-textColorStatusOffline: black;
+	qproperty-textColorStatusAway: gray;
+	qproperty-textColorStatusBusy: gray;
+	qproperty-textColorStatusOnline: darkGreen;
+	qproperty-textColorStatusInactive: gray;
+	qproperty-textColorGroup: rgb(123, 123, 123);
+}
+
+SearchDialog
+{
+	qproperty-textColorLocal: red;
+	qproperty-textColorDownloading: green;
+	qproperty-textColorNoSources: rgb(0, 0, 19);
+	qproperty-textColorLowSources: rgb(0, 0, 38);
+	qproperty-textColorHighSources: rgb(0, 0, 228);
+}
+
+RSTextBrowser, MimeTextEdit
+{
+	/*qproperty-textColorQuote: rgb(120, 153, 34);*/
+	qproperty-textColorQuotes: ColorList(#789922 #039bd5 #800000 #800080 #008080 #b10dc9 #85144b #3d9970);
+}
+
+/* OpModeStatus need to be at end to overload other values*/
+
+OpModeStatus {
+	qproperty-opMode_Full_Color: #CCFFCC;
+	qproperty-opMode_NoTurtle_Color: #CCCCFF;
+	qproperty-opMode_Gaming_Color: #FFFFCC;
+	qproperty-opMode_Minimal_Color: #FFCCCC;
+}
+OpModeStatus[opMode="Full"] {
+	color: black;
+	background: #CCFFCC;
+}
+OpModeStatus[opMode="NoTurtle"] {
+	color: black;
+	background: #CCCCFF;
+}
+OpModeStatus[opMode="Gaming"] {
+	color: black;
+	background: #FFFFCC;
+}
+OpModeStatus[opMode="Minimal"] {
+	color: black;
+	background: #FFCCCC;
+}
+
+/*Property Values at end to overwrite other settings*/
+
+[new=false] {
+	/*background: rgba(248,248,248,75%);*/
+}
+[new=true] {
+	background: rgba(220,236,253,75%);
+}
+
+[valid=true] {
+	background: rgba(0,255,0,15%);
+}
+
+[valid=false] {
+	background: rgba(255,0,0,15%);
+}
+
+[WrongValue="true"] {
+	background-color: #FF8080;
+}
+
+/* HeaderFrame & TitleBarFrame */
+
+QFrame[objectName^="headerFrame"],
+QFrame[objectName^="headerBFrame"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #9BDBF9, stop:1 #1592CD);
+	border: 1px;
+	border-radius: 4px;
+}
+QFrame[objectName^="#headerBFrame"] {
+	border: 1px;
+	border-radius: 4px;
+}
+QFrame[objectName^="headerFrame"] > *:!hover,
+QFrame[objectName^="headerBFrame"] > *:!hover{
+	background: transparent;
+	color: white;
+}
+
+QFrame[objectName^="toolBarFrame"],
+QFrame[objectName^="toolBarFrameTop"] {
+	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #FEFEFE, stop:1 #E8E8E8);
+	border: 1px solid #CCCCCC;
+}
+
+ChatWidget QFrame#titleBarFrame {
```

Comment:
This use generic "titleBarFrame" name. If not generic, rename it.

Reply:
else everything get the same color, i wanted only for the chatwidget titleBarFrame the color